src/miner.h

```diff
@@ -31,4 +31,12 @@ struct CMinerStatus
 extern CMinerStatus MinerStatus;
 extern unsigned int nMinerSleep;
 
+namespace supercfwd
+{
+    int MaybeRequest();
+    void HashResponseHook(CNode* fromNode, const std::string& neural_response);
+    void QuorumResponseHook(CNode* fromNode, const std::string& neural_response);
+    void SendResponse(CNode* fromNode, const std::string& req_hash);
+}
+
```

Comment:
There is no apparent reason to keep this in miner. Can it be moved to another file?

Reply:
Yes it can be moved, and probably should. I put it in miner, because it was related to adding neural contracts to mined blocks. If you are talking about the declarations being in the .h file, they are there because the functions are defined in the .cpp file.